rfd/0050-join-methods.md

```diff
@@ -0,0 +1,109 @@
+---
+authors: Nic Klaassen <nic@goteleport.com>
+state: Draft
+---
+
+# RFD 50 - Cluster Join Methods and Endpoints
+
+## What
+
+There are multiple ways a node can get credentials for a Teleport cluster. This
+document describes the current node joining methods and details upcoming changes
+for the IAM join method (RFD 41) and Certificate Bot (RFD YY).
+
+## Why
+
+This RFD can serve as a reference for all of the ways Teleport nodes can get
+cluster credentials in order to facilitate discussion of the current methods and
+new endpoints that will be added for the IAM join method and Certificate Bot.
+
+## Details
+
+When "joining" a cluster, a Teleport node must authenticate itself using either
+a secret token or one of the new AWS join methods (EC2 or IAM method). If the
+node can successfully authenticate, the Auth server will return signed SSH and
```

Comment:
For clarity, I would just add a note here that the secret token is only used on the first-time connect and from that point on the signed certificates are used.